backends/gdx-backend-dragome/src/com/badlogic/gdx/backends/dragome/js/storage/Storage.java

```diff
@@ -28,11 +29,11 @@
 	static final boolean localStorageSupported = checkStorageSupport("localStorage");
 	static final boolean sessionStorageSupported = checkStorageSupport("sessionStorage");
 
-	public static Storage getLocalStorageIfSupported () {
-		Storage storage = null;
+	public static ArrayBufferFactory getLocalStorageIfSupported () {
+		ArrayBufferFactory storage = null;
 		if (localStorageSupported || sessionStorageSupported) {
 			Object instance = ScriptHelper.eval("window.localStorage", null);
-			storage = JsDelegateFactory.createFrom(instance, Storage.class);
+			storage = JsDelegateFactory.createFrom(instance, ArrayBufferFactory.class);
 		}
 		return storage;

```

Comment:
Storage (https://www.w3.org/TR/webstorage/#storage-0) is a impl not yet complete. I let there for experimenting before I actually finish it when using with libgdx.  Why changed Storage.class to ArrayBufferFactory.class ?